sdk/src/main/java/com/qonversion/android/sdk/internal/user/controller/UserControllerImpl.kt

```diff
@@ -55,6 +67,40 @@ internal class UserControllerImpl(
         }
     }
 
+    override fun onAppForeground(isFirst: Boolean) {
+        if (!isFirst) {
+            return
+        }
+
+        scope.launch {
+            try {
+                if (userCacher.getActual() !== null) {
+                    return@launch
+                }
+
+                val userId = userDataStorage.requireUserId()
+                val newUser = userService.getUser(userId)
+                handleNewUserInfo(newUser)
+            } catch (exception: QonversionException) {
+                logger.error("Requesting user on app first foreground failed", exception)
```

Comment:
I think it's better to use `logger.info` here because even the error occurred, a user doesn't know anything about this request. The user didn't trigger this request, so he shouldn't know about this error in the log.